### PR TITLE
remove fatals and exits

### PR DESCRIPTION
**Pull request by rhs**

### CODE REVIEW

File: cmd/kubewatch/kubewatch.go [hunk 1]
```diff
@@ -149,12 +149,12 @@ func (s *Syncer) serve() {
 	ln, err := net.Listen("tcp", fmt.Sprintf(":%s", PORT))
 	if err != nil {
 		// Error starting or closing listener:
-		log.Fatalf("kubewatch: %v", err)
+		panic(fmt.Sprintf("kubewatch: %v", err))
```
Comment (ark3) [inline cmd/kubewatch/kubewatch.go]: There's also `log.Panicf(...)`

Comment (ark3) [reply cmd/kubewatch/kubewatch.go]: Also, why is this better?

File: cmd/teleproxy/teleproxy.go [hunk 2]
```diff
@@ -98,9 +102,9 @@ func main() {
 		// do nothing
 	case VERSION:
 		fmt.Println("teleproxy", "version", Version)
-		os.Exit(0)
+		return 0
 	default:
-		log.Fatalf("TPY: unrecognized mode: %v", *mode)
+		panic(fmt.Sprintf("TPY: unrecognized mode: %v", *mode))
```
Comment (ark3) [inline cmd/teleproxy/teleproxy.go]: Why is this better? Am I missing a `recover` somewhere?

File: cmd/watt/main.go [hunk 3]
```diff
@@ -125,12 +130,14 @@ func runWatt(cmd *cobra.Command, args []string) {
...
+		return 1
 	}
+
+	return 0
 }
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.Fatalln(err)
+		log.Println(err)
```
Comment (ark3) [inline cmd/watt/main.go]: Shouldn't you still `os.Exit(1)` in this case? I think the original might have been fine here.

File: cmd/kubewatch/kubewatch.go (hunk 1 above)
Comment (rhs) [reply cmd/kubewatch/kubewatch.go]: Ah, didn't know about that... thanks!

**Comment by rhs** [comment]
I should also note, that kubewatch is no longer used, so the changes there are mostly to eliminate noise from grep.

**Pull request merged**